grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -76,5 +76,12 @@ class NewConsentGroupController extends AuthenticatedController {
         }
     }
 
+    def approveConsentGroup() {
+        Gson gson = new Gson()
+        Object input = gson.fromJson(gson.toJson(request.JSON), Object.class)
+        String projectKey = params.id
+        Issue issue = queryService.findByKey(projectKey)
+        Issue updatedIssue = issueService.modifyIssueProperties(issue, input)
```

Comment:
You're right for both comments. I used try catch block, and returned updatedIssue as response. Sounds ok for you? Though this will be used in my next PR for general approve, so it might come back :sweat_smile:

Reply:
Sounds good!